### ISSUE TITLE
Information about the dataset

### ISSUE THREAD

**Issue by arpitg1304** [themightyoarfish/deepVO#2]
Hi,

Can you please tell us about the structure of your dataset like what are the sequences , what is chunk size and all the parameters used to create the batches. It is bit confusing to understand without looking at the dataset. It would be really helpful if you could tell us about the structure of your data

Thanks

**Comment by themightyoarfish** [comment]
The dataset consisted of 6-d poses and consecutive camera images. We cut the entire sequence into chunks of some length (the `sequence_length`) and each pose within a subsequence was relative to the initial pose of the subsequence (not the initial pose of the robot, but the data manager will do that for you). We tried different subsequence lengths but I can't give you a good value. We managed any kind of convergence only on a test dataset where the robot moved only forwards and backwards randomly.
